zipline/assets/_assets.pyx

```diff
@@ -203,6 +203,26 @@ cdef class Equity(Asset):
         params = ', '.join(strings)
         return 'Equity(%d, %s)' % (self.sid, params)
 
+    property security_start_date:
+        """
+        DEPRECATION: This property should be deprecated and is only present for
```

Comment:
I'm unfamiliar with deprecation in general - is there some way that we could make calls to security_start_date throw deprecation warnings in to the user log?